charts/consul/templates/api-gateway-controller-deployment.yaml

```diff
@@ -57,6 +57,7 @@ spec:
       containers:
       - name: api-gateway-controller
         image: {{ .Values.apiGateway.image }}
+        imagePullPolicy: {{ .Values.global.imagePullPolicy }}
```

Comment:
here is the default behavior: https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting

Reply:
i have not used any default value coz that setting defaults will be the job of global imagePullPolicy.

if thats not configured it will behave according to the default polices that @thisisnotashwin  mentioned